crates/consensus/beacon/src/engine/mod.rs

```diff
@@ -907,10 +907,8 @@ where
 
         #[allow(clippy::single_match)]
         match &error {
-            Error::Execution(
-                error @ BlockExecutionError::Validation(BlockValidationError::BlockPreMerge {
-                    ..
-                }),
+            Error::Canonical(
```

Comment:
@mattsse I think I spotted the bug, this one had to be modified too. Now it should be good